src/OrchardCore.Modules/OrchardCore.Html/Settings/HtmlBodyPartSettings.cs

```diff
@@ -2,5 +2,6 @@ namespace OrchardCore.Html.Settings
 {
     public class HtmlBodyPartSettings
     {
+        public bool AllowCustomScripts { get; set; }
     }
```

Comment:
This probably needs to be a site setting, not an individual setting

before you add to the field as well ;)

Reply:
We said it should be on individualthings

Reply:
ok fine with me